src/Templates/parts/nav.twig

```diff
@@ -44,6 +39,11 @@
             </ul>
           </li>
         {% endif %}
+        {% if websiteurl %}
+        <li class="nav-img">
+          <a href="{{websiteurl}}"><img alt="MyRadio" src="{{baseurl}}img/{{shortname}}.svg"></a>
```

Comment:
I will update the alt text. Yeah I get that alt text is for accessibility, but having a another element or otherwise being shown for non accessiblity reasons.